test/internal/helpers/Player.java

```diff
@@ -192,6 +192,42 @@ public static Cleanups setStats(int muscle, int mysticality, int moxie) {
     return new Cleanups(() -> setStats(0, 0, 0));
   }
 
+  public static Cleanups setMuscle(int muscle, int buffedMuscle) {
```

Comment:
I feel like KoLCharacter should have methods for setting each of Muscle, Myst and Moxie that get called here, instead of calling these methods that use the existing stat values.

Reply:
I don't know if it makes any real sense to do this other than for tests - we never see isolated stats in KoL - they always come as one big dump of info.

Reply:
yes, but tests are also code, and this has the opportunity to simplify a bit of test logic (and make tests that only want to set one stat) at the cost of three extra method calls each time `KoLCharacter.setStatPoints` gets called. I think it's worthwhile.

Reply:
Ok!